### DISCUSSION TITLE
GitHub Education Account Verification

### DISCUSSION THREAD

**Discussion by nabroleonx** [community/community#55047] [GitHub Education]
### Hi everyone,

 I am writing to request your assistance in verifying my eligibility for the GitHub Education Pack. As a dedicated university student, I am eager to take advantage of the resources and benefits offered through this program. I would like to explain my situation regarding the email verification process. Unfortunately, my university does not provide email addresses to students, which poses a challenge when attempting to verify my status as a student. Instead, I have been using my personal Gmail account for all my academic and professional needs.
 
 I have seen some other guy having the same [issues](https://github.com/orgs/community/discussions/53695). However, it hasn't been resolved.

I understand that the email verification process is in place to ensure that only eligible students can access the Package. However, I assure you that I am a currently enrolled university student, and I can provide any necessary documentation or proof of enrollment to support my claim.

I kindly request that you consider alternative methods for verifying my student status, such as grade reports, student IDs, or any other form of documentation that would demonstrate my current affiliation with a recognized educational institution.

Being a part of the GitHub Education Pack would greatly benefit me in my studies and future career endeavors. I appreciate your understanding and assistance in this matter.

Hope to hear from you soon

**Comment by samus-aran** [comment]
Hi @nabroleonx 👋 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

**Reply by nabroleonx** [reply]
@samus-aran Thanks for the response. I have a student ID but the thing is my application got rejected because I am not using a university issued email. However as I have tried to explain, our university doesn't give out emails to students. 

Would you mind explaining how I can get around this issue?
